bin/spark-submit.sh

```diff
@@ -15,14 +15,14 @@ spark-submit \
 --conf spark.executor.extraJavaOptions=-XX:+UseCompressedOops \
 --conf spark.executor.extraJavaOptions="-Dlog4j.configuration=file://$SPARK_HOME/conf/log4j.properties" \
 --conf spark.driver.extraJavaOptions="-Dlog4j.configuration=file://$SPARK_HOME/conf/log4j.properties" \
---conf spark.kryo.referenceTracking=false \
+--conf spark.kryo.referenceTracking=true \
 --conf spark.network.timeout=18000s \
 --conf spark.executor.heartbeatInterval=12000s \
 --conf spark.dynamicAllocation.executorIdleTimeout=12000s \
 --conf spark.port.maxRetries=200 \
---packages org.apache.spark:spark-avro_2.11:2.4.0 \
+--packages org.apache.spark:spark-avro_2.12:3.1.2 \
```

Comment:
@dattranm This will not work for Spark 2.4.x

Do you have a suggestion how to adapt spark_submit.sh depending on the Spark version used?